kolibri/plugins/learn/assets/src/views/LibraryPage/UnPinnedDevices.vue

```diff
@@ -62,9 +75,21 @@
           color: this.$themeTokens.text,
           marginBottom: `${this.windowGutter}px`,
           minHeight: `${this.overallHeight}px`,
+          textAlign: 'center',
         };
       },
     },
+    methods: {
+      getDeviceIcon(device) {
```

Comment:
As this component only handles one device at a time, this could be a computed prop instead, referencing `this.device`